packages/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { FC, useState } from 'react';
+import cn from 'classnames';
+
+import {
+  Breadcrumb,
+  BreadcrumbLinkProps,
+  BreadcrumbsProps,
+} from './Breadcrumbs.types';
+
+import styles from './styles/default.module.css';
+
+const DefaultLink: React.FC<BreadcrumbLinkProps> = ({
+  link,
+  className,
+  children,
+  ...rest
+}) => (
+  <a className={className} href={link} {...rest}>
+    {children}
+  </a>
+);
+
+const Breadcrumbs: FC<BreadcrumbsProps> = ({
+  breadcrumbs,
+  dataTestId,
+  className,
+}) => {
+  if (breadcrumbs.length === 0) {
+    return null;
+  }
+
+  const Link = DefaultLink as (props: BreadcrumbLinkProps) => JSX.Element;
+
+  const firstBreadcrumb = breadcrumbs[0];
+  const lastBreadcrumb = breadcrumbs[breadcrumbs.length - 1];
+  const hiddenBreadcrumbs = breadcrumbs.slice(1, -1);
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks, @typescript-eslint/naming-convention
+  const [showHiddenBreadcrumbs, setShowHiddenBreadcrumbs] = useState(false);
```

Comment:
hooks in react need to run every time a component runs. So you need to move this above your `if` block with the return null (line 28). Then you can remove the eslint-disable